world.js

```diff
@@ -781,7 +782,31 @@ const _connectRoom = async (roomName, worldURL) => {
 
 const objects = [];
 world.getObjects = () => objects;
-world.addObject = (contentId, parentId = null, position = new THREE.Vector3(), quaternion = new THREE.Quaternion(), options = {}) => {
+world.addObject = async (contentId, parentId = null, position = new THREE.Vector3(), quaternion = new THREE.Quaternion(), options = {}) => {
+  (async () => {
+    const token = await contracts.sidechain.NFT.methods.tokenByIdFull(parseInt(contentId)).call();
+    const monetizationPointer = await contracts.sidechain.Account.methods.getMetadata(token.owner, 'monetizationPointer').call();
+    if (monetizationPointer && document.monetization && !document.querySelector("meta[name=monetization]")) {
+      const monetizationTag = document.createElement('meta');
+      monetizationTag.name = 'monetization';
+      monetizationTag.content = monetizationPointer;
+      document.head.appendChild(monetizationTag);
```

Comment:
This tag should be removed when the object is removed.

Reply:
0559424